task-2/src/Inventory.ts

```diff
@@ -0,0 +1,21 @@
+import { Item } from "./Item";
+import { ItemComparator } from "./ItemComparator";
+
+export class Inventory {
+    private items: Item[];
+
+    addItem(item: Item): void {
+        this.items.push(item);
+    }
+
+    sort(comparator?: ItemComparator): void {
+        const defaultComparator = function (a: Item, b: Item): number {
```

Comment:
I think you can move this function outside of the class. It's independent and does not use any scope variables